GenderPayGap.WebUI/Models/AddOrganisation/AddOrganisationManualViewModel.cs

```diff
@@ -31,18 +34,44 @@ public class AddOrganisationManualViewModel : GovUkViewModel
 
         public string Query { get; set; }
 
+        [GovUkValidateCharacterCount(MaxCharacters = 100)]
+        [GovUkDisplayNameForErrors(NameAtStartOfSentence = "Employer name", NameWithinSentence = "employer name")]

```

Comment:
Sorry to ask for this but can we invert the order of these two lines to match the pattern in the rest of the file? (i.e. name for errors first, then character limit after)

Reply:
Good spot! Thanks